src/data.py

```diff
@@ -93,12 +112,19 @@ def parse_payment_methods(methods):
   return payment_methods
 
 def parse_operating_hours(eatery, dining_items):
+  """Returns a list of OperatingHoursTypes for each dining event for an eatery.
+  Calls parse_events to populate the events field for OperatingHoursType.
+  Args:
+      eatery (dict): A valid json segment from Cornell Dining that contains eatery information
+      dining_items (list): A list that holds a dictionary for the items an eatery serves and a flag
+       for healthy options
+  """
   new_operating_hours = []
   hours_list = eatery['operatingHours']
   for hours in hours_list:
     new_date = hours.get('date', '')
     hours_events = hours['events']
-
+    #merge the dining items from the json (for non-dining hall eateries) with the menu for an event
```

Comment:
Let's add a space between the '#' and 'merge'!

Reply:
Added it back!